src/typechain/IGlpManager.d.ts

```diff
@@ -0,0 +1,124 @@
+/* Autogenerated file. Do not edit manually. */
```

Comment:
@ioanaroceanu2000 @0xZenus is there a way to avoid the need to have all of these typechain files stored in the sdk codebase?

Reply:
I don't know of nay other solution

Reply:
yes, one solution is to add just the json abi with the used functions (a lot of gain in the bundle size) but the contract calls will not be typed. We can aim to do this at some point but it requires some effort.